net/respondd/README.md

```diff
@@ -0,0 +1,35 @@
+respondd providers are C modules (shared objects). These modules should include
+<respondd.h>, which provides the following definitions:
```

Comment:
"<json-c/json.h> and <respondd.h>, the latter of which provides the following definitions:"